### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #1058: Extract configuration of `JavaLibraryPlugin` to `buildSrc/`
Follow-up to https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1031

### PR DESCRIPTION
**Pull request by MarconZet**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Some hooks use `grep`. The macOS version of `grep` (FreeBSD) differs from GNU `g
 In order to make `grep` and eventually the hooks working one must:
 1. Install `grep` via `brew` (it will not override system's `grep` &mdash; it can be executed as `ggrep`)
 2. Run `brew ls -v grep`; among the other a path like should be found `/opt/homebrew/Cellar/grep/<grep-version>/libexec/gnubin/grep`
-3. Prepend the found path without `/grep` suffix to `PATH` (`/opt/homebrew/Cellar/grep/<grep-version>/libexec/gnubin` in that case).
+3. Prepend the found path without `/grep` suffix to `PATH` (`/usr/local/Cellar/grep/3.8_1/libexec/gnubin` in that case).
```
[the remark this answers is not in the document]
Comment (PawelLipski) [reply CONTRIBUTING.md:76]: Yes, pls list the possible options in point `2.`, then in point `3.` just show one specific example

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -50,6 +50,7 @@ pluginPackages-spotless = "com.diffplug.spotless:spotless-plugin-gradle:6.10.0"
 pluginPackages-taskTree = "com.dorongold.plugins:task-tree:2.1.0"
 pluginPackages-versionCatalogUpdate = "nl.littlerobots.version-catalog-update:nl.littlerobots.version-catalog-update.gradle.plugin:0.5.3"
 pluginPackages-versionsFilter = "se.ascp.gradle:gradle-versions-filter:0.1.16"
+pluginPackages-javaLibrary = "???"
```
Comment (PawelLipski) [inline gradle/libs.versions.toml:53]: `javaLibrary` is a bundled/built-in plugin, shipped together with Gradle itself; I'd guess there isn't a need to include it in version catalog... or is there? 🤔

Comment (MarconZet) [reply gradle/libs.versions.toml:53]: JavaLibrary is built-in, but I can't find the DSL

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -72,7 +72,8 @@ This is because one may not be emulating bash environment in any way or doing it
 Some hooks use `grep`. The macOS version of `grep` (FreeBSD) differs from GNU `grep`.
 In order to make `grep` and eventually the hooks working one must:
 1. Install `grep` via `brew` (it will not override system's `grep` &mdash; it can be executed as `ggrep`)
-2. Run `brew ls -v grep`; among the other a path like should be found `/opt/homebrew/Cellar/grep/<grep-version>/libexec/gnubin/grep`
+2. Run `brew ls -v grep`; among the other a path like this should be found `/opt/homebrew/Cellar/grep/<grep-version>/libexec/gnubin/grep`
```
Comment (PawelLipski) [inline CONTRIBUTING.md:75]: Ok so IIUC this is hard to get right, pls just extract this update to CONTRIBUTING.md and we're good to go.

Btw have you experienced

Comment (mkondratek) [reply CONTRIBUTING.md:75]: a bit, yeah. recently I updated this line b/c there was a specific `<grep-version>` given and I was unable to commit (with Git hooks) after updating grep 😆

Comment (mkondratek) [reply CONTRIBUTING.md:75]: @PawelLipski, what is your proposition `pls just extract this update to CONTRIBUTING.md` about? we are within CONTRIBUTING.md 🤔

**Review by mkondratek — APPROVED** [review]
lgtm; but pls make sure that the thing mentioned by :pawełeł: is resolved

File: CONTRIBUTING.md (hunk 3 above)
Comment (PawelLipski) [reply CONTRIBUTING.md:75]: I should've rather written: just keep the changes to CONTIRIBUTING.md, even if the rest of the changes (to Gradle stuff) is to be discarded 😅

**Pull request closed without merging**